extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -25,38 +25,102 @@ import {EventType, dispatch} from './events';
 import {Services} from '../../../src/services';
 import {addAttributesToElement, closest, matches} from '../../../src/dom';
 import {createShadowRootWithStyle, getSourceOriginForElement} from './utils';
-import {dev, devAssert, user} from '../../../src/log';
+import {dev, devAssert, user, userAssert} from '../../../src/log';
 import {dict} from '../../../src/utils/object';
 import {getAmpdoc} from '../../../src/service';
 import {htmlFor, htmlRefs} from '../../../src/static-template';
 import {isProtocolValid, parseUrlDeprecated} from '../../../src/url';
 import {resetStyles, setImportantStyles, toggle} from '../../../src/style';
 
 /**
- * Enum of elements that can be expanded.
+ * Action icons to be placed in tooltip.
  * @enum {string}
+ * @private
+ */
+const ActionIcon = {
+  LAUNCH: 'i-amphtml-tooltip-action-icon-launch',
+  EXPAND: 'i-amphtml-tooltip-action-icon-expand',
+};
+
+/**
+ * Components that can be expanded.
+ * @const {!Object}
+ * @private
  */
 const EXPANDABLE_COMPONENTS = {
-  TWITTER: 'amp-twitter',
+  'amp-twitter': {
+    componentIcon: 'data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.' +
+    'w3.org/2000/svg" width="400" height="400"><g fill="none" fill-rule="eve' +
+    'nodd"><path d="M0 0h400v400H0z"/><path fill="%231da1f2" fill-rule="nonz' +
+    'ero" d="M153.62 301.59c94.34 0 145.94-78.16 145.94-145.94 0-2.22 0-4.43' +
+    '-.15-6.63A104.36 104.36 0 0 0 325 122.47a102.38 102.38 0 0 1-29.46 8.07 ' +
+    '51.47 51.47 0 0 0 22.55-28.37 102.79 102.79 0 0 1-32.57 12.45c-15.9-16.' +
+    '906-41.163-21.044-61.625-10.093-20.461 10.95-31.032 34.266-25.785 56.87' +
+    '3A145.62 145.62 0 0 1 92.4 107.81c-13.614 23.436-6.66 53.419 15.88 68.4' +
+    '7A50.91 50.91 0 0 1 85 169.86v.65c.007 24.416 17.218 45.445 41.15 50.28' +
+    'a51.21 51.21 0 0 1-23.16.88c6.72 20.894 25.976 35.208 47.92 35.62a102.9' +
+    '2 102.92 0 0 1-63.7 22 104.41 104.41 0 0 1-12.21-.74 145.21 145.21 0 0 ' +
+    '0 78.62 23"/></g></svg>',
```

Comment:
Nit: Can we keep these in the CSS with the other launch icons? This will get big, especially as we start allowing many many components